pages/blog/new-asyncapi-tools-page.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'New Tools Dashboard for AsyncAPI'
+date: 2023-01-30T06:00:00+01:00
+type: Communication
+tags:
+  - Automation
+  - JSON Schema
+cover: /img/posts/new-asyncapi-tools-page/cover.webp
```

Comment:
Really struggling with seeing an apple product (soooo enterprise) for this blog post... i almost feel like it would be more interesting if we used [a picture of tools such as this one](https://www.dewalt.com/products/hand-tools) ... what do you think 🧐 💭 ?

![image](https://user-images.githubusercontent.com/19964402/215650034-cfda72b7-b04b-4f27-9115-15c1611c04bf.png)